### ISSUE TITLE
Extend Kyma cli with new features to access various kyma components

### ISSUE THREAD

**Issue by rakesh-garimella** [kyma-project/cli#477]
**Description**

Currently using Kyma cli we can provision, install kyma as a next step to have more feature rich kyma cli we should extend it to have support for various components currently part of kyma.
eg.
```
kyma lambda create
```
or 
```
kyma event list
```
```
kyma servicecatalog list
```

such kind of feature can help user to access kyma via cli (also hide the complicated grep, sed pipe magic) this can be helpful specially in pipelines if they want to have some automation around kyma. Also this can be used as a stepping stone for users who are new to Kubernetes but they might find the cli helpful to get aquainted to all the features kyma offers.

while implementing we should also look into adding support for these components as plugins so that the enduser has options what all plugins he/she wants.

**Comment by pbochynski** [comment]
Can you start with commands to create and deploy serverless function and with defining an event trigger? This seems to be the most wanted feature.

**Comment by kwiatekus** [comment]
Here is the issue for function kick started part: https://github.com/kyma-project/cli/issues/551

**Comment by a-thaler** [comment]
Thanks for your feedback on where the CLI should develop to. I will take it as input for defining the future direction of the project and derive some roadmap page out of it. That ticket is quite high level without a concrete milestone to reach and I prefer to close it.
Indeed the implementation of commands for serverless has started and is planned to be released with 1.17 release
